Code/wwutil/stackdump-linux.cpp

```diff
@@ -0,0 +1,6 @@
+#include "stackdump.h"
+
+void cStackDump::Print_Call_Stack(void)
+{
+    // do nothing
```

Comment:
[libunwind](https://www.nongnu.org/libunwind/man/libunwind(3).html) can provide this information.

So perhaps implement this (when libunwind is available), or add a fixme (+create ticket)?

Reply:
I was actually planning to use [`std::basic_stacktrace`](https://en.cppreference.com/w/cpp/utility/basic_stacktrace.html), but that requires C++23.  I believe that the API is the same as the boost version though.

Reply:
My long-term thoughts are to remove this and just use one implementation for both, so that's really why it's blank.  Creating a ticket is a good idea though, I will do that

Reply:
For what it's worth, in SDL's internal test library, we're using imagehlp for Windows, and libunwind for all other platforms.